environment.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: [:cellphone],
+          items: %i[medipack chevy_versa]
+        },
+        standard_transportation_pack: {
+          intelligence: %i[cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry chevy_versa]
+        },
+        simple_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry financial]
+        },
+        standard_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: %i[remington_870 colt_1911 machete hatchet],
+          items: %i[handcuffs medipack chemistry financial]
+        }
+      }
+    end
+  end
+
+  class Control
+    attr_accessor :missions
+
+    def initialize
+      @missions = {}
+    end
+
+    def new_mission(name:, objective:, pack:)
+      @missions[name] = {
+        objective: objective,
+        pack: pack,
+        status: :active
+      }
+    end
+
+    statuses = %w[paused aborted failed accomplished]
+    statuses.each do |status|
+      define_method("set_mission_to_#{status}") do |arg|
+        @missions[arg][:status] = status.to_sym
+      end
+    end
+  end
+
+  class Human
+    attr_accessor :id, :name, :personal_data, :professional_data
+
+    def initialize(name:)
+      @id = object_id
+      @name = name
+    end
+
+    data = %w[personal_data professional_data]
+    data.each do |type|
+      define_method("set_#{type}") do |arg|
+        instance_variable_set("@#{type}", arg)
+      end
+    end
+  end
+
+  class Worker < Human
+    attr_accessor :standard_shift, :extra_shift
+
+    def initialize(name:)
+      super name
+      @standard_shift = {
+        id: @id,
+        hours: 8,
+        payment: 8,
+        facility: String.new,
+        status: nil
+      }
```

Comment:
Same thing here, I would set this hash like:
`@standard_shift = { id: @id, hours: 8, payment: 8, facility: '', status: nil }`